pages/common/git-info.md

```diff
@@ -0,0 +1,13 @@
+# git info
+
+> Display repository information.
```

Comment:
Should we be specific and mention a git repo.
I understand its under git so of course it implies a git repo. but still .. thoughts ?

```suggestion
> Display Git repository information.
```